### ISSUE TITLE
Audit of VA.gov pages with tables

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#30815]
## Issue Description

As part of the tables/lists accessibility work we are starting, we need to do an audit of our pages to find where we use tables. Once we do, we need to determine if they are valid tables per Josh Kim's guidance:

- **Non-matrix tables** with only one header type (either column or row) should be written as lists with headings. For example, the tables on survivor dic rates should be lists. 

- **Matrix tables** with both column and row headers should be written as tables.
For example, the special monthly compensation rates tables should remain as tables as it's easier for screen reader users to parse through. They will first identify the row relevant to them, and then navigate right to determine the values.

Once we do the audit, we should separate the pages with valid tables from the ones that need to be redone. If there's any uncertainty, we can confirm with Josh.

## Tasks
- [ ] Do a search of our pages for tables
- [ ] List the pages that have valid tables
- [ ] List the pages that have tables that need to be redone

## Acceptance Criteria
- [ ] All pages with tables have been accounted for and a list has been created

**Comment by RLHecht** [comment]
Note: When doing this audit, we need to make sure that this housing page is on the list of ones to change out of table form and into list form: https://www.va.gov/housing-assistance/home-loans/eligibility/

**Comment by cmaeng** [comment]
@RLHecht here's a download of nodes with tables. creating the functionality to do this is real time will take some effort but if you ever need a refreshed list or if this isn't what you're looking for just let us know. [Uploading... Nodes with tables.csv]

**Comment by cmaeng** [comment]
retrying:
[Nodes with tables.csv](https://github.com/department-of-veterans-affairs/va.gov-team/files/8689387/Nodes.with.tables.csv)

**Comment by RLHecht** [comment]
Update: The CMS team is looking into their audit because there are a few pages that have tables that didn't show up in their original list.

**Comment by RLHecht** [comment]
Moving to blocked until CMS team is done with their audit.

**Comment by bethpottsVADEPO** [comment]
@RLHecht Could we pick this back up so we know the # of tables in our benefit hubs and R&S articles that need to be reworked?

**Comment by laflannery** [comment]
[Google sheet for table audit](https://docs.google.com/spreadsheets/d/1n6zQWMzbYnLV-cqDeSjBWA4Lv84u0TV_fcndhkSgW9s/edit#gid=802180051)

**Comment by RLHecht** [comment]
@SarahKay8 I highlighted in green in Laura's spreadsheet the pages we already have tickets for to address the tables. Do you want to review the other ones to determine what work needs to be done? I think some we would need to remove the tables and others may need table headers. Thank you!

**Comment by SarahKay8** [comment]
Yep will review!

**Comment by SarahKay8** [comment]
@RLHecht The **What your decision review or appeal status means** and **How to find out if you should get a higher TSGLI payment** tables visually have tabular relationships. Each include a relationship of each cell to its column header which are necessary for understanding. I think using tables with headers, captions and scope attribute is okay for this content.

**Comment by RLHecht** [comment]
@SarahKay8 following up from our conversation in standup yesterday, you'll want to look at all the tables that are listed on this spreadsheet, regardless of what section is listed. I hid any tables that our team isn't responsible for so the rest of them are on pages we own. Thanks!

**Comment by SarahKay8** [comment]
@RLHecht gotcha! Thanks for the clarification

**Comment by SarahKay8** [comment]
@RLHecht I would keep the non-green highlighted tables as tables.  However, things like “Primary care services ” for the 2023 outpatient care copay rates tables (etc.), I would consider as a row header for its subordinate information. But that’ll be hard to do in HTML without changing how the table looks. 
We could do it in a PDF,  in the tags, (for future references if we ever need to)  but because these are web pages. I have nothing additional to add on my end.

**Comment by SarahKay8** [comment]
After further discussion and research with the rest of the a11y team, I highlighted a few more tables that are being used as a layout that would be better off as a list with headers. So far, I have rows 567-574
104-108
14-20 highlighted in green on our excel sheet. Will update with comments as I continue my second audit @RLHecht

**Comment by SarahKay8** [comment]
@RLHecht I highlighted cell #971 in the excel. Other than that, I have no additional tables to add.

**Comment by RLHecht** [comment]
Closing this out. Will create new tickets for the the content team to rework pages with tables.
